docs/en/Architecture-DataFlow.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: global
+title: Architecture and Data Flow
+group: Home
+---
+
+* Table of Contents
+{:toc}
+
+## Architecture
+
+### Overview
+
+Alluxio is not a persistent storage system. Instead,
+Alluxio serves as a new data access layer in the ecosystem, residing between any persistent storage
+systems (such as Amazon S3, Microsoft Azure Object Store, Apache HDFS or OpenStack Swift)
+and computation frameworks
+(such as Apache Spark, Presto or Hadoop MapReduce).
+There are multiple benefits to have this layer in the stack:
+
+- For user applications and computation frameworks, Alluxio can provide them fast storage, facilitating data sharing and locality between jobs, regardless of whether they are running on the same computation engine. As a result, Alluxio can serve the data at memory speed when data is local, or the computation cluster network speed when data is in Alluxio. Data is only read once from the under storage system on the first time it’s accessed. Therefore, the data access can be significantly accelerated when the access to the under storage is not fast. To achieve the best performance, Alluxio is recommended
```

Comment:
can we split these long lines into shorter lines? We follow that style in most of our markdown docs, and it makes it easier to read diffs and write review comments

Reply:
done